### PR TITLE
Update docs with more examples and updated help

### PR DESCRIPTION
**Pull request by sideangleside**
Updated README.md to include many more examples.

### CODE REVIEW

**Comment by sideangleside** [comment]
Bah. Don't merge this PR. I forgot we have `EXAMPLES.md` with additional examples

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Network Classic and get it registered to Foreman & Katello.
 * Installing subscription-manager and its pre-reqs (updated yum & openssl)
 * Make an API call to Katello to create the Foreman Host associated with the user specified Org/Location
 * Install the Katello consumer RPM
-* Running rhn-migrate-classic-to-rhsm (with the user provisded activation key) to get product certs on a system
+* Running rhn-migrate-classic-to-rhsm (with the user provisioned activation key) to get product certs on a system
```
Comment (evgeni) [inline README.md]: I think this should have been "provided"?

File: README.md [hunk 2]
```diff
@@ -98,21 +98,236 @@ When using the `--legacy-purge` option, a user account on the legacy environment
...
+There are times where you wish to not install puppet, perhaps you have a differing or existing configuration management system.
+
+~~~
+# ./bootstrap.py -l admin \
+    -s foreman.example.com \
+    -o "Red Hat" \
+    -L RDU \
+    -g "RHEL7/Crash" \
+    -a ak-Reg_To_Crash \
+    --skip-puppet
```
Comment (evgeni) [inline README.md:126]: `--skip puppet`, please :)

File: README.md [hunk 3]
```diff
@@ -98,21 +98,236 @@ When using the `--legacy-purge` option, a user account on the legacy environment
...
+**NOTES**
+
+ - the `--skip-foreman` switch implies `--skip-puppet`
+ - When using `--skip-foreman`, it is expected that the organization specified  (via `--organization|-o`) is specified via **LABEL**, not **NAME**.
+
+~~~
+# ./bootstrap.py -s foreman.example.com \
+    -a ak_Reg_To_Dev_EL7 \
+    -o "Red_Hat" \
+    --skip-foreman
```
Comment (evgeni) [inline README.md:142]: `--skip foreman`, please :)

File: README.md [hunk 4]
```diff
@@ -98,21 +98,236 @@ When using the `--legacy-purge` option, a user account on the legacy environment
...
+It is recommended to set which repositories that you want enabled on your activation keys via the UI or via `hammer activation-key product-content`. However, older versions of `subscription-manager` (versions < 1.10) do not support product content overrides. The `--enablerepos` switch accepts a comma separated lists of repositories that are passed to `subscription-manager` that will be enabled at registration time.
+
+~~~
+# ./bootstrap.py -l admin \
+    -s foreman.example.com \
+    -o "Red Hat" \
+    -L RDU \
+    -g "RHEL7/Crash" \
+    -a ak-Reg_To_Crash \
+    --enablerepos=rhel-7-server-extras-rpms, rhel-7-server-optional-rpms
```
Comment (evgeni) [inline README.md]: Dat space will break it!

**Comment by evgeni** [comment]
Can you check that "Puppet" is always spelled "Puppet", not "puppet" in the text parts, and "Foreman" and "bootstrap.py" (not "Bootstrap.py"?)

**Review by evgeni — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -98,21 +98,236 @@ When using the `--legacy-purge` option, a user account on the legacy environment
 * system group administrator for a system group that the system is a member of
 * granted permissions to the system explicitly via Users-> account-> 'Systems Administered by this User'
 
-# Usage:
+# Usages:
 
+### Registering a system to Foreman
```
Comment (evgeni) [inline README.md]: Why Foreman and not "Foreman + Katello" as before? (just cuerious)

File: README.md (hunk 1 above)
Comment (sideangleside) [reply README.md]: Yes. Damn autocorrect.

File: README.md (hunk 2 above)
Comment (sideangleside) [reply README.md:126]: I didn't want to retroactively remove the old examples. They are still honored, so I'd rather have them documented. (Cause users will ask).

File: README.md (hunk 3 above)
Comment (sideangleside) [reply README.md:142]: I didn't want to retroactively remove the old examples. They are still honored, so I'd rather have them documented. (Cause users will ask).

File: README.md (hunk 5 above)
Comment (sideangleside) [reply README.md]: inconsistent behavior is inconsistent

**Review by evgeni — APPROVED** [review]

**Pull request merged**